docs/DeveloperGuide.adoc

```diff
@@ -729,12 +729,82 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |use shorter aliases for commands|enter commands quickly and not have to type in the full command
+
+|`* * *` |user |find people by their tags|view people under certain tags
+
+|`* * *` |user |have a responsive inbuilt browser with similar response times to external browsers|use the inbuilt browser smoothly
+
+|`* * *` |user |add tags cumulatively|edit tags conveniently
+
+|`* * *` |user |add a person with fewer parameters|add someone I don't know all the details of
+
+|`* * *` |user |edit contact details|modify contacts without having to delete the contact
+
+|`* * *` |user |view in-line help via the help command|view the help without having to navigate the user guide(which is not CLI friendly)
+
+|`* * *` |user |add contacts with multiple phone numbers|have contact entries with multiple phone numbers without the need for multiple entries
+
+|`* * *` |user |revert to a previous version of my AddressBook|restore from a backup if my contact data is accidentally lost
+
+|`* * *` |user |navigate the navigation using only my keyboard (using preset keybindings)|use the application solely with my keyboard, as with CLI-focused apps
+
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |user |have Google Contacts integration link:#google-contacts-api[Google Contacts API]| view and modify my contacts on other platforms than my computer
+
+|`* *` |user |have a reminder system tag to names| remember my appointments with other people
+
+|`* *` |user |access a person's Facebook account via in the in-built browser|use Facebook features from the AddressBook
+
+|`* *` |user |find a subset of contacts using specified parameters|filter through my contacts
+
+|`* *` |user |click on the address to be shown its Google Maps result|easily navigate to my contact's location
```

Comment:
`click on the address to be shown its Google Maps result` sound more of a use case. Try instead `locate a person's address on Google Maps`.